crates/bevy_ui/src/focus.rs

```diff
@@ -177,16 +198,31 @@ pub fn ui_focus_system(
                 let ui_position = position.truncate();
                 let extents = node.node.size() / 2.0;
                 let mut min = ui_position - extents;
-                let mut max = ui_position + extents;
                 if let Some(clip) = node.calculated_clip {
                     min = Vec2::max(min, clip.clip.min);
-                    max = Vec2::min(max, clip.clip.max);
                 }
-                // if the current cursor position is within the bounds of the node, consider it for
+
+                // The mouse position relative to the node
+                // (0., 0.) is the bottom-left corner, (1., 1.) is the upper-right corner
```

Comment:
considering the ui coordinate uses 0,0 to mean top-left I think the cursor should keep this convention.

Reply:
I checked it and it seems it already is that way. I don't know why, but I wrote the opposite in the comment.